searx/utils.py

```diff
@@ -39,12 +39,14 @@
 _ECMA_UNESCAPE2_RE = re.compile(r'%([0-9a-fA-F]{2})', re.UNICODE)
 
 _STORAGE_UNIT_VALUE: Dict[str, int] = {
-    'TB': 1024 * 1024 * 1024 * 1024,
-    'GB': 1024 * 1024 * 1024,
-    'MB': 1024 * 1024,
-    'TiB': 1000 * 1000 * 1000 * 1000,
-    'MiB': 1000 * 1000,
-    'KiB': 1000,
+    'TB': 1000 * 1000 * 1000 * 1000,
+    'GB': 1000 * 1000 * 1000,
+    'MB': 1000 * 1000,
+    'kB': 1000,
```

Comment:
`k` in lower case: is it intended ?

Reply:
On base 10^n (metric/decimal) it is ``kB`` on base 2^n (binary) it is ``KB``--> https://en.wikipedia.org/wiki/Kilobyte

Since we are talking about bytes I prefer a 2^10 (1024/binary) factor and the ``KB`` notation.